### PR TITLE
py-svgpath: new port

### PR DESCRIPTION
**Pull request by FlyingSamson**
#### Description

This adds the python package `svg.path` which offers functionality to parse, manipulate, and work with SVG path objects.

###### Tested on
macOS 13.6.4 22G513 arm64
Xcode 15.2 15C500b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-svgpath/Portfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-svgpath
+version             6.3
+revision            0
+
+python.rootname     svg.path
```
Comment (reneeotten) [inline python/py-svgpath/Portfile:10]: please move this directly under "name"

File: python/py-svgpath/Portfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+checksums           rmd160  bb83fad1fc8bd5132bf3fbf352cea2666c63d686 \
+                    sha256  e937740a316a7fec86acd217ab6226e112f51328078524126bb7ea9dbe7b1ade \
+                    size    20920
+
+maintainers         {@flyingsamson tuxcad.de:flyingsamson} openmaintainer
+description         SVG path objects and parser
+long_description    svg.path is a collection of objects that implement the different path \
+                    commands in SVG, and a parser for SVG path definitions.
+license             MIT
+
```
Comment (reneeotten) [inline python/py-svgpath/Portfile:21]: this package appears to be only Python and nothing gets compiled. A good way to check is to check the download files on PyPI. In this case the .whl file contains "none-any". 
That means it installs the same files on all OS versions and is architecture independent. 

To state these things in the Portfile one would add:

supported_archs noarch
platforms {darwin any}

**Review by reneeotten — CHANGES_REQUESTED** [review]
@FlyingSamson thanks for the new port. One minor request out of convention in addition to a few lines that should be added.

File: python/py-svgpath/Portfile (hunk 1 above)
Comment (FlyingSamson) [reply python/py-svgpath/Portfile]: Done

File: python/py-svgpath/Portfile (hunk 2 above)
Comment (FlyingSamson) [reply python/py-svgpath/Portfile:22]: Ah, thanks for pointing that out, and explaining what to look out for!

**Review by reneeotten — APPROVED** [review]

**Comment by reneeotten** [comment]
thanks @FlyingSamson !

**Pull request merged**